PDSim/core/StateFlooded.pxd

```diff
@@ -0,0 +1,60 @@
+from libcpp.string cimport string
+import cython
+cimport cython
+
+from libcpp.vector cimport vector
+
+include "AbstractState.pxd"
+    
+cdef extern from "Python.h":
+    char* __FILE__
+
+cdef extern from "Python.h":
+    int __LINE__
+
+
+
+
+"""
+#In Cython we would do something equivalent to CoolProp.pxd 
+# This class inheritate from State_Flooded which calculates the mixture properties 
+# and returns the values in a similar way of State class in CoolProp.pxd
+"""
+
+#TODO:  check consistency with StateFlooded.pyx   get_h_m or get_h. Maybe we should differentiate from standard State class and use _m to indicate mixture properties
+
+cdef class State_Flooded:
```

Comment:
Also, better to name StateFlooded - that is more standard.  Class definitions should be CamelCase, without underscores.